operator/src/main/resources/scripts/livenessProbe.sh

```diff
@@ -91,6 +91,16 @@ if [ -f ${STATEFILE} ] && [ `grep -c "FAILED_NOT_RESTARTABLE" ${STATEFILE}` -eq
   exit $RETVAL
 fi
 
+if [ -x ${LIVENESS_PROBE_CUSTOM_SCRIPT} ]; then
+  $LIVENESS_PROBE_CUSTOM_SCRIPT
+elif [ -O ${LIVENESS_PROBE_CUSTOM_SCRIPT} ]; then
+  chmod 770 $LIVENESS_PROBE_CUSTOM_SCRIPT && $LIVENESS_PROBE_CUSTOM_SCRIPT
```

Comment:
I suspect this might be a little tough for an end user to debug?  If it isn't, maybe we can provide an option to append output to a rotating log file?   Or maybe we should track this separately and provide such an option for the entire liveness probe script - not just this part?  @rjeberhard thoughts?